src/component/common/PermissionButton/PermissionButton.tsx

```diff
@@ -68,11 +68,11 @@ const PermissionButton: React.FC<IPermissionButtonProps> = ({
             title={formatAccessText(access, tooltipProps?.title)}
             arrow
         >
-            <span id={id}>
+            <span id={id} role="tooltip">
                 <Button
                     onClick={onClick}
                     disabled={disabled || !access}
-                    aria-describedby={id}
+                    aria-labelledby={id}
```

Comment:
Can you reference a tooltip with "aria-labelledby"? According  to [the states and properties section of MDN's docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tooltip_Role#associated_wai-aria_roles_states_and_properties):

> The element that triggers the tooltip references the tooltip element with [aria-describedby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby).